src/hooks/use-url-routing.js

```diff
@@ -80,6 +80,19 @@ export default function useUrlRouting(options) {
     [countSlideElements, isSlideElementOutOfBounds, isSlideOutOfBounds]
   );
 
+  const goToSlide = React.useCallback(
+    slideNumber => {
+      const qs = queryString.stringify({
+        presenterMode: currentPresenterMode || undefined,
+        immediate: true,
+        slide: slideNumber,
+        slideElement: 0
```

Comment:
This should be `DEFAULT_SLIDE_ELEMENT_INDEX`. A value of `0` would show the first `Appear` item instead of giving the presenter a chance to animate it in.